serverless/cmd/sequence/main.go

```diff
@@ -39,13 +39,27 @@ var (
 
 func main() {
 	flag.Parse()
+	var err error
```

Comment:
No need to move this line - it's used below by the `if *create / else` stuff - it's to have the `err` _outside_ the scope of the if so we just have a single `if err != nil` check which covers both cases (on L75).

Here, though, you're declare-and-assign'ing an `err` which exists only in the scope of the if (`if err := ...`), consequently we don't need the var declaration.

(This is why you're getting the warning from the `LGTM` check on this PR, BTW.)